Make.inc

```diff
@@ -160,6 +160,9 @@ BUILD_LLDB = 0
 # Path to cmake (override in Make.user if needed)
 CMAKE ?= cmake
 
+# Point pkg-config to only look at our libraries
+export PKG_CONFIG_LIBDIR = $(JULIAHOME)/usr/lib/pkgconfig
+
```

Comment:
Does this work?  http://stackoverflow.com/questions/23843106/how-to-set-environment-variable-in-makefile